frontstage/views/account/account.py

```diff
@@ -49,6 +68,7 @@ def change_account_details(session):
                                                       attributes_changed,
                                                       respondent_details,
                                                       update_required_flag)
+        email_update_required_flag = form['email_address'].data != respondent_details['emailAddress']
```

Comment:
Can you change all instances of the following variables in this file:
- `email_update_required_flag` to `is_email_update_required`
- `update_required_flag` to `is_update_required`

Its minor change but I think language is important.

Reply:
👍